pkg/server/catalog/catalog.go

```diff
@@ -180,6 +195,64 @@ func Load(ctx context.Context, config Config) (_ *Repository, err error) {
 	return repo, nil
 }
 
+func ValidateConfig(ctx context.Context, config Config) (pluginNotes map[string][]string, err error) {
+	if c, ok := config.PluginConfigs.Find(nodeAttestorType, jointoken.PluginName); ok && c.IsEnabled() && c.IsExternal() {
+		return nil, errors.New("the built-in join_token node attestor cannot be overridden by an external plugin")
+	}
+
+	log, _ := test.NewNullLogger()
```

Comment:
can we replace this with no test code?

Reply:
Yes, we can.  We already set `config.Log` to log to `/dev/null` so we don't actually need this test logger. I've also switched it to use `io.Discard` which should work in more places.